tokio-util/src/task/spawn_aborting.rs

```diff
@@ -0,0 +1,38 @@
+use std::ops::{Deref, DerefMut};
+
+use tokio::task::JoinHandle;
+
+use futures_core::Future;
+
+/// This is a wrapper type around JoinHandle that allows it to be dropped.
```

Comment:
A `JoinHandle` can already be dropped, just like all other Rust values. So, this documentation, while technically correct, isn't really explaining what this type is actually for. How about something more like this:
```suggestion
/// A wrapper around a [`tokio::task::JoinHandle`] which [aborts] the task
/// automatically when the handle is  dropped.
///
/// [aborts]: tokio::task::JoinHandle::abort
```